contracts/BondedSortitionPool.sol

```diff
@@ -17,21 +17,27 @@ import "./DynamicArray.sol";
 /// checked and, if necessary, updated in the sortition pool. If the changes
 /// would be detrimental to the operator, the operator selection is performed
 /// again with the updated input to ensure correctness.
+/// The pool should specify a reasonable minimum bond for operators trying to
+/// join the pool, to prevent griefing by operators joining without enough
+/// bondable value.
 contract BondedSortitionPool is AbstractSortitionPool {
     using DynamicArray for DynamicArray.UintArray;
     using DynamicArray for DynamicArray.AddressArray;
     using RNG for RNG.State;
-    // The pool should specify a reasonable minimum bond
-    // for operators trying to join the pool,
-    // to prevent griefing by operators joining without enough bondable value.
-    // After we start selecting groups
-    // this value can be set to equal the most recent request's bondValue.
+
 
     struct PoolParams {
         IStaking stakingContract;
         uint256 minimumStake;
         IBonding bondingContract;
-        uint256 minimumBondableValue;
+        // Defines the minimum unbounded value the operator needs to have to be
+        // eligible to join and stay in the sortition pool. Operators not
+        // satisfying minimum bond value are removed from the poool.
+        uint256 minimumBond;
+        // Bond required from each operator for the currently pending group
+        // selection. If operator does not have at least this unbounded value,
+        // it is skipped during the selection.
+        uint256 selectionBond;
```

Comment:
```suggestion
        uint256 requiredBond;
```

Reply:
I am not sure about it. `minimumBond` is also a required bond. What if we do: `minimumBond -> minimumBondableValue` and leave `selectionBond` as-is since it's a bond required for the current selection?

Reply:
Please see https://github.com/keep-network/sortition-pools/pull/84/commits/d09133cd125317052b516afc632abe9ceecf459a. I have also cleaned up all comments to make the distincion about `minimumBondableValue` and `selectionBond` clear and to use `minimum bondable value` consistently.

Reply:
> I am not sure about it. `minimumBond` is also a required bond. 

You're right. I should have proposed `requestedBond`. If you think `selectionBond` is better than `requestedBond` I'm fine with leaving it as is.

Reply:
I think they are equally fine but if you prefer `requestedBond` it means we do have a preference and I'll rename it :wink:

Reply:
In the beginning `selectionBond` confused me for a while. I believe `requestedBond` would be easier to understand.

Reply:
Rename time, then!